### PR TITLE
Fix restoration and activation of help widgets

### LINKED ISSUE #2619: Help pages do not restore when the app is refreshed.
Open a help reference page -> refresh the app -> help page not restored.

### PR DESCRIPTION
**Pull request by blink1073**
Fixes #2619.

### CODE REVIEW

File: packages/help-extension/src/index.ts [hunk 1]
```diff
@@ -145,11 +145,24 @@ class HelpWidget extends Widget {
...
   /**
+   * The url of the widget.
+   */
+  readonly url: string;
+
+  /**
+   * Handle activate requests for the widget.
+   */
+  protected onActivateRequest(msg: Message): void {
+    this.node.tabIndex = -1;
```
Comment (sccolbert) [inline packages/help-extension/src/index.ts]: shouldn't this be set in the constructor?

**Comment by sccolbert** [comment]
How does this fix the resoration?

**Comment by blink1073** [comment]
The restoration relied on a `url` property on the widget, which we had lost when we wrapped the iframe widget originally.

File: packages/help-extension/src/index.ts (hunk 1 above)
Comment (blink1073) [inline packages/help-extension/src/index.ts]: This is the convention we had been using.

**Pull request merged**